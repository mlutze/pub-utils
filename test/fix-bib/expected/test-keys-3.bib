@article{brachthauser2022,
  title  = {Effects, Capabilities, and Boxes: From Scope-Based Reasoning to Type-Based Reasoning and Back},
  author = {Jonathan Immanuel Brachth√§user and Philipp Schuster and Edward Lee and Aleksander Boruch-Gruszecki},
  year   = {2022}
}
