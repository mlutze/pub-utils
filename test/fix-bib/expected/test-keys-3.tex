\documentclass{article}

\begin{document}
\cite{brachthauser2022}
\end{document}